commands/account/createOverride.ts

```diff
@@ -0,0 +1,75 @@
+import fs from 'fs-extra';
+import path from 'path';
+import { Argv, ArgumentsCamelCase } from 'yargs';
+import { getCwd } from '@hubspot/local-dev-lib/path';
+import { logger } from '@hubspot/local-dev-lib/logger';
+import { getConfigPath, getAccountId } from '@hubspot/local-dev-lib/config';
+import { addConfigOptions } from '../../lib/commonOpts';
+import { i18n } from '../../lib/lang';
+import { EXIT_CODES } from '../../lib/enums/exitCodes';
+import { selectAccountFromConfig } from '../../lib/prompts/accountsPrompt';
+import { CommonArgs, ConfigArgs } from '../../types/Yargs';
+
+const i18nKey = 'commands.account.subcommands.createOverride';
+
+export const describe = null; // i18n(`${i18nKey}.describe`);
+
+export const command = 'create-override [account]';
+
+type AccountInfoArgs = CommonArgs &
```

Comment:
Looks like a copy/paste leftover here 😄

Reply:
Not quite sure what you mean--I based the type off of the types in the `hs account info` command but had to make some changes. Should I move this to `Types`, so we can reuse between commands?

Reply:
Addressed in https://github.com/HubSpot/hubspot-cli/pull/1349/commits/3cb9c4b8c3e2f171fc96e70836a9c644a3df8f1f